src/index.html

```diff
@@ -17,16 +17,22 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <input
+        type="text"
+        data-qa="keypress"
```

Comment:
The `data-qa` attribute is set to `keypress`, but it should be `search-bar-large` according to the task requirements.